torch/distributed/optim/utils.py

```diff
@@ -24,6 +24,18 @@
     optim.Adamax: _FunctionalAdamax,
 }
 
+
+# Interface to insert a new functional optimizer to functional_optim_map
+# Example::
+#     >>> import the new functional optimizer
+#     >>> from xyz import new_optimizer
+#     >>> from torch.distributed.optim.utils import register_functional_optim
+#     >>>     register_functional_optim(new_optimizer,new_optimizer)
```

Comment:
the mapping should be optimizer to the functional implementation, right?

Reply:
Yep - this is just to show case that it must be the new functional optim to be registered and use it as the key as well. I will update it to key, optim to be uniform